junction/proposals/templatetags/proposal_filters.py

```diff
@@ -22,3 +22,8 @@ def get_content_urls(proposal):
         return urls
     else:
         return []
+        
```

Comment:
We follow [pep 8](https://www.python.org/dev/peps/pep-0008/). PEP 8 advised to leave two lines after every function. You can integrate flake 8 with emacs.